conda-recipe/meta.yaml

```diff
@@ -18,7 +18,8 @@ requirements:
       - wheel
     build:
       - {{ compiler('cxx') }}
-      - {{ compiler('dpcpp') }}  >=2023.0  # [not osx]
+      - {{ compiler('dpcpp') }}  =2023.1.0  # [not osx]
```

Comment:
What is the purpose of this hard pinning?

Reply:
Unblock internal CI loop until compiler issue is resolved. It's a kind of possible temporary solution, instead of muting all tests which are crashing now.